docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Transaction decoding will no longer fail when encountering spam or invalid ERC20 tokens.
```

Comment:
This does not seem correct. I have invalid tokens and my decoding does not break. Something else breaks? So perhaps rephrase this? 

```suggestion
* :bug:`-` Transaction decoding will no longer fail when encountering spam or invalid ERC20 tokens.
```